airflow/www/decorators.py

```diff
@@ -56,6 +57,7 @@ def wrapper(*args, **kwargs):
                     for k, v in chain(request.values.items(multi=True), request.view_args.items())
                     if k not in fields_skip_logging
                 ]
+                extra_fields = [(k, secrets_masker.redact(v, k)) for k, v in extra_fields]
```

Comment:
Why not redact "in place" as we build up (on line 55/56)?

Reply:
sounds like ~the same overhead and nice sepration of concerns if you ask me

Reply:
I guess we are never talking anything but a "small" number of items anyway